source/text/vss-strings.ads

```diff
@@ -98,6 +98,241 @@ package VSS.Strings is
       Right : Virtual_String) return Boolean;
    --  Compare two strings in binary order of code points.
 
+   --  function "&"
+   --    (Left  : Virtual_String;
+   --     Right : Virtual_String) return Virtual_String;
+   --  function "&"
+   --    (Left  : Virtual_String;
+   --     Right : VSS.Characters.Virtual_Character) return Virtual_String;
+   --  function "&"
+   --    (Left  : VSS.Characters.Virtual_Character;
+   --     Right : Virtual_String) return Virtual_String;
+   --  function "&"
+   --    (Left  : VSS.Characters.Virtual_Character;
+   --     Right : VSS.Characters.Virtual_Character) return Virtual_String;
+   --  --  Concatenation operator for virtual string and virtual characters.
+
+   procedure Clear (Self : in out Virtual_String'Class);
+   --  Remove all data.
+
+   procedure Append
+     (Self : in out Virtual_String'Class;
+      Item : VSS.Characters.Virtual_Character);
+   --  Append given abstract character to the end of the string.
+
+   --  function Append
```

Comment:
Why do we need Append functions if we have "&"?

Reply:
Why not? It is alternative name for completeness.

Reply:
* It makes API more complex, 
* Append isn't a good name for it, may be Concatenate is better, but I would avoid it at all